src/util/dsp/LogSlider.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+namespace otto::util::dsp {
+  class LogSlider
+  {
+  public:
+      LogSlider();
+      ~LogSlider();
+
+      double proportionOfLengthToValue(double proportion);
```

Comment:
functions and variables use `lower_snake_case`, mainly for consistency with the standard